lms/djangoapps/courseware/url_helpers.py

```diff
@@ -9,19 +9,23 @@
 from xmodule.modulestore.django import modulestore
 
 
-def get_redirect_url(course_key, usage_key):
+def get_redirect_url(course_key, usage_key, unified_course_view=False):
```

Comment:
@dianakhuang It is probably fine as-is because it is going to be deleted, but in the off chance that the clean-up work doesn't happen for a long time, you could also make this work the way @andy-armstrong mentioned.  If you change it, just add a note like `Remember to remove the unused request parameter when removing the waffle flag` to the `TODO` comment.